.teamcity/settings.kts

```diff
@@ -438,9 +438,17 @@ object CheckCodeStyle : BuildType({
 				. "${'$'}NVM_DIR/nvm.sh" --no-use
 				nvm install
 
-				# Code style
-				FILES_TO_LINT=${'$'}(git diff --name-only --diff-filter=d refs/remotes/origin/master...HEAD | grep -E '^(client/|packages/|apps/)' | grep -E '\.[jt]sx?${'$'}' || exit 0)
+				# Find files to lint
+				if [[ "%teamcity.build.branch.is_default%" == "true" ]]; then
+					FILES_TO_LINT="."
+				else
+					FILES_TO_LINT=${'$'}(git diff --name-only --diff-filter=d refs/remotes/origin/master...HEAD | grep -E '(\.[jt]sx?|\.md)${'$'}' || exit 0)
```

Comment:
cc @saramarcondes just heads up to keep in mind this will be one more thing to rename once we rename the primary branch to `trunk`.

Reply:
TeamCity has `%teamcity.build.branch%`, but I think that points to `/refs/heads/master`, not `refs/remotes/origin/master`. Not sure how that will affect this command.